packages/iota/src/highLevel/sendAdvanced.ts

```diff
@@ -149,7 +159,7 @@ export function buildTransactionPayload(
             const o: IBasicOutput = {
                 type: BASIC_OUTPUT_TYPE,
                 amount: output.amount.toString(),
-                nativeTokens: [],
+                nativeTokens: output.nativeTokens,
                 unlockConditions: [
```

Comment:
Looks like the unlock conditions are hardcoded here.  Sending outputs with additional unlock conditions, other than Address unlock condition, will also get discarded. 
Looks like the whole sendAdvanced function needs a refactor :)